kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,18 +1,22 @@
 return {
   fields = {
-    timeout = {type = "number", default = 60000, required = true },
-    keepalive = {type = "number", default = 60000, required = true },
-    aws_key = {type = "string", required = true},
-    aws_secret = {type = "string", required = true},
-    aws_region = {type = "string", required = true, enum = {
+    timeout = { type = "number", default = 60000, required = true },
```

Comment:
So it's okay when new code is in different style with the existing? I'm just trying to understand the pattern I should follow to pass the code review